src/Types/AccountInfo.cs

```diff
@@ -18,19 +18,62 @@ namespace Concordium.Sdk.Types;
 /// Not null if and only if the account is a baker or delegator. In that case
 /// it is the information about the baker or delegator.
 /// </param>
+/// <param name="Schedule">
+/// Release schedule for any locked up amount. This could be an empty
+/// release schedule.
+/// </param>
+/// <param name="Cooldowns">
+/// The stake on the account that is in cooldown.
+/// There can be multiple amounts in cooldown that expire at different times.
+/// This was introduced in protocol version 7, and so is not present in
+/// earlier protocol versions.
+/// </param>
+/// <param name="AvailableBalance">
+/// The available (unencrypted) balance of the account (i.e. that can be transferred
+/// or used to pay for transactions). This is the balance minus the locked amount.
+/// The locked amount is the maximum of the amount in the release schedule and
+/// the total amount that is actively staked or in cooldown (inactive stake).
```

Comment:
You can stake amounts that are in release schedule, which is why we don't add them.

Reply:
I see. So that means if I have already some amounts of funds on a release schedule I start staking with that account. It is considered, that I start staking first the funds that are already looked in the release schedule, and when that is not enough start staking my "normal" funds on that account.